packages/wix-ui-tpa/src/components/Image/Image.tsx

```diff
@@ -129,6 +150,12 @@ export class Image extends React.Component<ImageProps> {
         })}
         data-hook={this.props['data-hook']}
       >
+        {isError && (
+          <div className={classes.errorMessage} aria-hidden="true">
+            <ErrorIcon className={classes.errorIcon} />
+            <Text>{errorMessage}</Text>
```

Comment:
I think we can use the second approach to style the text:
```suggestion
            <Text className={classes.errorMessage}>{errorMessage}</Text>
```

And in the .st file we can have something like:
```css
.root .errorMessage {
  -st-extends: TPAText;
  -st-mixin: TPAText(MainTextColor value(errorTextColor));
}
```

Similar to what Calendar does:
https://github.com/wix/wix-style-react/blob/master/packages/wix-ui-tpa/src/components/CalendarCell/CalendarCell.st.css#L133-L139


But then we should rename the selector we pass to the wrapper div (including inside the .st file):
```jsx
<div className={classes.errorWrapper} aria-hidden="true">
```